### DISCUSSION TITLE
Blank screen when open vue

### DISCUSSION THREAD

**Discussion by DidVer** [eclipse/windowbuilder#165] [Q&A] [answered]
When i open a view in eclipse, no desing screen is present and no toole for desing the windows
i re installation of eclipse and pluging but same result
this view was open in windows buider à few week before
[vClasseurVirtuel.zip](https://github.com/eclipse/windowbuilder/files/7985682/vClasseurVirtuel.zip)

**Comment by wimjongman** [comment]
What version of windowbuilder are you using?

**Reply by wimjongman** [reply]
And what version of Java?

**Reply by DidVer** [reply]
Version 1.97 and replace by 1.9.6 for test 
eclipse 2021-12
java 8 updaez 321 (64)
java se dev kit 16.0.2,

**Comment by wimjongman** [comment, marked as answer]
Please install from Marketplace the latest version 1.9.8 or higher

**Reply by DidVer** [reply]
i try this version this afternoon

**Comment by DidVer** [comment]
I have installed 1.9.8 but message when parsing to find jframe. With the modification to find the jframe, the message is always present. I have created a new window file and transfere the old source code to the new file - > OK the parsing created the design and the tools are in the screen. I think that the format of the old file was corrupt because with the same source it's OK on the new file
